Exercises/filling2.cpp

```diff
@@ -7,41 +7,80 @@
 
 typedef boost::adjacency_list<boost::vecS, boost::vecS, boost::undirectedS> Graph;
 
+enum error_types {
```

Comment:
Better to declare it in .h file so users can use it as well. The naming convention for enums is ALL_CAPITALS without leading underscore: EDGES, NEGATIVE_VERTICES, CONTINUITY, VERTICES.